ui/src/hooks/useFileManage.ts

```diff
@@ -159,6 +177,10 @@ export const useFileManage = () => {
       mittBus.on('file-refresh', () => {
         refresh(<WebSocket>socket);
       });
+
+      mittBus.on('change-path', (path: string) => {
+        handleChangePath(<WebSocket>socket, path);
+      });
     }
   }
 
```

Comment:
There is no difference between original file management code from the previous year's dataset and the updated one from 2024 because it seems that there may have been some changes made to this code but with insignificant modifications or adjustments, as the code appears to work correctly without any errors or noticeable deviations.